<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>general_settings.title</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/mainapp/sources/appico.ico</normaloff>:/mainapp/sources/appico.ico</iconset>
  </property>
  <widget class="QCheckBox" name="enableDebugMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>general_settings.debug_mode</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="maxAdrBookLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>general_settings.max_adr_book.label</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxAdrBook">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
   <property name="value">
    <number>4096</number>
   </property>
  </widget>
  <widget class="QLabel" name="maxAccountLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>general_settings.max_accounts.label</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxAccount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
   <property name="value">
    <number>4096</number>
   </property>
  </widget>
  <widget class="QLabel" name="serversLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>general_settings.servers.label</string>
   </property>
  </widget>
  <widget class="QListWidget" name="hostsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>host.b32.i2p</string>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/mainapp/sources/i2p.png</normaloff>:/mainapp/sources/i2p.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addHost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rmHost">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editHost">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>general_settings.edit_host</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applySettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>general_settings.apply_settings_button</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
